IS7024/Pages/HollywoodBowl.cshtml

```diff
@@ -8,7 +8,7 @@
 
 <div class="text-center">
     <h1 class="display-4">Upcoming Events at the Hollywood Bowl</h1>
-    <hr>
+    <hr />
```

Comment:
This change can be ignored, it is not necessary as Horizontal Rule does not require an end tag. Sorry!

Reply:
No need to apologize... you are correct.  Adding the / is XHTML compliant.